ui/app/models/database/role.js

```diff
@@ -45,8 +45,9 @@ export default Model.extend({
   rotation_period: attr({
     editType: 'ttl',
     defaultValue: '24h',
-    subText:
+    helperTextDisabled:
```

Comment:
Do any other models' attributes with `editType: 'ttl',` have `subtext`? If so we should switch them out

Reply:
Nope these were the only TTL cases of `subText`. 

`aws-credential.js` does have a `helpText` attribute but I was planning to address that in another PR so this change could be merged. I was planning to set up AWS in order to test that TTL